include/z64actor.h

```diff
@@ -100,34 +100,34 @@ typedef struct {
     /* 0x18 */ Vec3f feetPos[2]; // Update by using `Actor_SetFeetPos` in PostLimbDraw
 } ActorShape; // size = 0x30
 
-#define ACTOR_FLAG_0 (1 << 0)
-#define ACTOR_FLAG_2 (1 << 2)
-#define ACTOR_FLAG_3 (1 << 3)
-#define ACTOR_FLAG_4 (1 << 4)
-#define ACTOR_FLAG_5 (1 << 5)
-#define ACTOR_FLAG_6 (1 << 6)
-#define ACTOR_FLAG_7 (1 << 7)
-#define ACTOR_FLAG_8 (1 << 8)
-#define ACTOR_FLAG_9 (1 << 9)
-#define ACTOR_FLAG_10 (1 << 10)
-#define ACTOR_FLAG_ENKUSA_CUT (1 << 11)
-#define ACTOR_FLAG_12 (1 << 12)
-#define ACTOR_FLAG_13 (1 << 13)
-#define ACTOR_FLAG_14 (1 << 14)
-#define ACTOR_FLAG_15 (1 << 15)
-#define ACTOR_FLAG_16 (1 << 16)
-#define ACTOR_FLAG_17 (1 << 17)
-#define ACTOR_FLAG_18 (1 << 18)
-#define ACTOR_FLAG_19 (1 << 19)
-#define ACTOR_FLAG_20 (1 << 20)
-#define ACTOR_FLAG_21 (1 << 21)
-#define ACTOR_FLAG_22 (1 << 22)
-#define ACTOR_FLAG_23 (1 << 23)
-#define ACTOR_FLAG_24 (1 << 24)
-#define ACTOR_FLAG_25 (1 << 25)
-#define ACTOR_FLAG_26 (1 << 26)
-#define ACTOR_FLAG_27 (1 << 27)
-#define ACTOR_FLAG_28 (1 << 28)
+#define ACTOR_FLAG_TARGETABLE (1 << 0) // Navi will fly over the actor and it can be Z targeted
+#define ACTOR_FLAG_UNFRIENDLY (1 << 2) // changes targeting behavior for unfriendly actors (sound, links stance)
+#define ACTOR_FLAG_FRIENDLY (1 << 3) // opposite of the unfriendly flag. flag is not checked against in the original game
+#define ACTOR_FLAG_NO_UPDATE_CULLING (1 << 4) // actor will keep updating even if outside of the uncull zone
+#define ACTOR_FLAG_NO_DRAW_CULLING (1 << 5) // actor will keep drawing even if outside of the uncull zone
+#define ACTOR_FLAG_NOT_CULLED (1 << 6) // actor is in the uncull zone. flag still updates regardless of the update/draw cull exclusion flags
```

Comment:
I would be fine with changing "uncull zone" to "active zone" in general, it may help with all the negative words going on here that make this system confusing. uncull zone is a name i was never happy with since the beginning.
in that case I would also change `ACTOR_FLAG_NOT_CULLED` -> `ACTOR_FLAG_IN_ACTIVE_ZONE`

Reply:
also, long conversation regarding these flags in general for anyone else who wants to talk about these https://discord.com/channels/688807550715560050/688851317593997489/935729171529887754

imo i would rather keep the first two as is, and rename the last one according to my suggestion above

Reply:
A summary of how these three flags work:

An actor is "culled" when it is outside the "uncull zone" defined by the three `uncullZone*` members in the `Actor` struct.

An actor being culled or not is indicated by `ACTOR_FLAG_6`/`ACTOR_FLAG_IN_UNCULL_ZONE` being set (actor is not culled) or cleared (actor is culled).

The set/clear state of this flag is updated every frame between actor update and actor draw (right before draw):
https://github.com/fig02/oot/blob/7774a67ea79076099820a6386a6a5b21c2ee5949/src/code/z_actor.c#L2421-L2425 :
```c
if (func_800314B0(globalCtx, actor)) {
    actor->flags |= ACTOR_FLAG_IN_UNCULL_ZONE;
} else {
    actor->flags &= ~ACTOR_FLAG_IN_UNCULL_ZONE;
}
```

When an actor is culled, by default its update and draw functions don't run.
This behavior can be overriden by setting the `ACTOR_FLAG_4`/`ACTOR_FLAG_NO_UPDATE_CULLING` and/or `ACTOR_FLAG_5`/`ACTOR_FLAG_NO_DRAW_CULLING` to make update and draw run regardless of the actor being culled.

-----------

Comments:

`ACTOR_FLAG_NO_UPDATE_CULLING`/draw one can't be "always update"/draw because frozen (`freezeTimer`) actors still don't update. This set of flag is "culling"-specific

"no X culling" means "do not apply culling for x" if that's clearer. Wording is hard, suggestions appreciated

Suggestions also appreciated on the "cull", "uncull" words if they can be replaced